{"name": "Создание пользователя. Негативный сценарий. Попытка повторного создания существующего клиента. ", "status": "passed", "description": "Создание нового пользователя не происходит, пользователь уже существует. Проверяется корректность кода ответа и сообщения об ошибке", "steps": [{"name": "Проверка статус-кода ответа", "status": "passed", "parameters": [{"name": "response", "value": "<Response [403]>"}, {"name": "expected_status_code", "value": "403"}], "start": 1719418239178, "stop": 1719418239178}, {"name": "Проверка значения в JSON по ключу", "status": "passed", "parameters": [{"name": "response", "value": "<Response [403]>"}, {"name": "name", "value": "'message'"}, {"name": "expected_value", "value": "'User already exists'"}, {"name": "error_message", "value": "'Error message is incorrect'"}], "start": 1719418239178, "stop": 1719418239178}], "start": 1719418239082, "stop": 1719418239178, "uuid": "3e63d1f1-d407-4515-9789-431ec10de06f", "historyId": "5805346c001ebd10bb9a91249c8f5c9d", "testCaseId": "5805346c001ebd10bb9a91249c8f5c9d", "fullName": "tests.test_create_user.TestCreateUser#test_create_user_already_exist", "labels": [{"name": "epic", "value": "Создание нового пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "DESKTOP-N0HG7C2"}, {"name": "thread", "value": "3456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}